spec/features/admin_categories_view_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe "the category view", type: :feature do
+
+  context "as an admin" do
+    before :each do
+      @category = FactoryGirl.create(:category)
+      @category2 = FactoryGirl.create(:category2)
+      @admin = FactoryGirl.create(:admin)
+      visit login_path
+      fill_in("session_email", with: @admin.email)
+      fill_in("session_password", with: @admin.password)
+      click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.